lively.morphic/rendering/renderer.js

```diff
@@ -161,13 +161,17 @@ export default class Renderer {
     }
 
     for (let morph of this.morphsWithStructuralChanges) {
-      this.renderStructuralChanges(morph);
+      morph.withAllSubmorphsDo(m => !this.renderMap.has(m) && this.renderMorph(m));
     }
 
     for (let morph of this.renderedMorphsWithChanges) {
       this.renderStylingChanges(morph);
     }
 
+    for (let morph of this.morphsWithStructuralChanges) {
+      this.renderStructuralChanges(morph);
+    }
+
```

Comment:
Why does this come after the "styling loop"?

Reply:
See above conversation.